### PR TITLE
Fix navigation for images

### PR DESCRIPTION
**Pull request by pbauer**
Can someone please review this? I am not sure if it is the right solution for https://github.com/plone/Products.CMFPlone/issues/830.

### CODE REVIEW

File: plone/app/layout/viewlets/common.py [hunk 1]
```diff
@@ -201,7 +201,8 @@ def update(self):
         self.selected_tabs = self.selectedTabs(portal_tabs=self.portal_tabs)
         self.selected_portal_tab = self.selected_tabs['portal']
 
-    def selectedTabs(self, default_tab='index_html', portal_tabs=()):
+    def selectedTabs(self, default_tab='index_html', portal_tabs=None):
+        portal_tabs = portal_tabs or []
```
Comment (mauritsvanrees) [inline plone/app/layout/viewlets/common.py]: I am guessing the original tuple was fine too, because it is immutable, but this sure looks safer.

**Comment by pbauer** [comment]
http://jenkins.plone.org/job/pull-request-5.0/223/

**Comment by mauritsvanrees** [comment]
Looks good to me. I tried it locally and it works.
Let's indeed see what Jenkins has to say.

File: plone/app/layout/viewlets/common.py (hunk 1 above)
Comment (pbauer) [inline plone/app/layout/viewlets/common.py]: True, I allways forget that tuples are immutable.

**Comment by pbauer** [comment]
http://jenkins.plone.org/job/pull-request-5.0/224/

**Comment by pbauer** [comment]
The tests pass now.

**Pull request merged**